cypress/integration/executions_spec.js

```diff
@@ -158,5 +158,51 @@ describe('Dashboard Executions Page', () => {
         });
       });
     });
+
+    it('should show an execution with limited information', () => {
+      const executionName = 'b313e777-d28a-435b-a0dd-f1fad08116t1';
+      const executionArn = 'arn:aws:states:us-east-1:596205514787:execution:TestSourceIntegrationIngestAndPublishGranuleStateMachine-yCAhWOss5Xgo:b313e777-d28a-435b-a0dd-f1fad08116t1';
+      const stateMachine = 'arn:aws:states:us-east-1:596205514787:stateMachine:TestSourceIntegrationIngestAndPublishGranuleStateMachine-yCAhWOss5Xgo';
+      cy.visit(`/#/executions/execution/${executionArn}`);
+
+      cy.contains('.heading--large', 'Execution');
+      cy.contains('.heading--medium', 'Visual workflow').should('not.exist');
+
+      const startMatch = fullDate('2018-12-06T19:18:11.174Z');
+      const endMatch = fullDate('2018-12-06T19:18:41.145Z');
+
+      cy.getFakeApiFixture(`executions/status/${executionArn}`).as('executionStatus');
+
+      cy.get('.status--process')
+      .within(() => {
+        cy.contains('Execution Status:').next().should('have.text', 'Succeeded');
+        cy.contains('Execution Arn:').next().should('have.text', executionArn);
+        cy.contains('State Machine Arn:').next().should('have.text', stateMachine);
+        cy.contains('Started:').next().should('have.text', startMatch);
+        cy.contains('Ended:').next().should('have.text', endMatch);
+
+        cy.get('@executionStatus').its('execution').then((execution) => {
+          cy.contains('Input:').next().find('pre').then(($content) =>
+            // parse and stringify JSON string to get the same format as in the fixture
+            expect(JSON.stringify(JSON.parse($content.text()))).to.eq(execution.input));
+
+          cy.contains('Input:').next().contains('Show Input').click();
```

Comment:
What do these two lines test?

Reply:
just to test the 'collapsible' elements are there.  I noticed the second 'click' doesn't work

Reply:
Is there any CSS class or something that we can check instead of clicking the elements?

Reply:
There is `.Collapsible` class, but `Hide Input` won't show up unless `Show Input` is clicked.  I removed the click on `Hide Input/Ouput`, I'm not sure why it doesn't work with cypress, but it's not necessary for this ticket.